<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LogDialog</class>
 <widget class="QDialog" name="LogDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>662</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1140</width>
    <height>16777215</height>
   </size>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QWidget" name="titlebar" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>870</width>
        <height>30</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>30</height>
       </size>
      </property>
      <layout class="QHBoxLayout" name="titlebar_layout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
      </layout>
     </widget>
     <widget class="QWidget" name="addWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>31</y>
        <width>870</width>
        <height>48</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <widget class="QLabel" name="label_24">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>5</y>
         <width>48</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>选择：</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBoxSelect">
       <property name="geometry">
        <rect>
         <x>60</x>
         <y>5</y>
         <width>331</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #e9eef2;
border-radius: 4px;
padding-left: 23px;
font-size: 14px;
color: rgba(0,0,0,0.8);
border: 0px;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="ButtonLoad">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>5</y>
         <width>80</width>
         <height>30</height>
        </rect>
       </property>
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>30</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background: #cce2f0;
border-radius: 4px;
font-size: 14px;</string>
       </property>
       <property name="text">
        <string>加载</string>
       </property>
       <property name="icon">
        <iconset resource="../../CHOWTAIFOOK_Detection.qrc">
         <normaloff>:/uiframe/images/viewtoolbar/import.svg</normaloff>:/uiframe/images/viewtoolbar/import.svg</iconset>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="mainWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>73</y>
        <width>870</width>
        <height>589</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Microsoft YaHei</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <widget class="Line" name="line">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>-60</y>
         <width>1141</width>
         <height>131</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QWidget" name="gridLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>871</width>
         <height>591</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="logLayout">
        <item row="0" column="0">
         <widget class="QTextBrowser" name="textBrowser"/>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../CHOWTAIFOOK_Detection.qrc"/>
 </resources>
 <connections/>
</ui>
